qemu/tests/cfg/block_scsi_generic_inquiry.cfg

```diff
@@ -0,0 +1,28 @@
+- block_scsi_generic_inquiry:
+    only virtio_blk
+    only Linux
+    only RHEL.7
```

Comment:
Why is limited to RHEL7, I see the fifth step of the case has a method to view rhel8?

Reply:
Correction, modify the steps after confirming with the function owner. Change case to only support rhel7.
limited is correct.